custom_components/powercalc/data/signify/LTC011/model.json

```
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly 1PM",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 2,
+        "VERSION": "master"
+    },
+    "name": "Hue ambiance ceiling",
```

You may set this to the marketed name. This will be shown in name column in supported model listing. https://github.com/bramstroker/homeassistant-powercalc/blob/master/docs/supported_models.md

```suggestion
    "name": "Hue Cher Ceiling Light",
```